cps-global-lib/README.md

```diff
@@ -13,18 +13,27 @@ Every workflow script in your Jenkins see these shared library scripts in their
 The directory structure of the shared library repository is as follows:
 
     (root)
-     +- src                 # groovy source files
-         +- org
-             +- foo
-                 +- Bar.groovy  # for org.foo.Bar class
-
+     +- src                     # groovy source files
+     |   +- org
+     |       +- foo
+     |           +- Bar.groovy  # for org.foo.Bar class
+     +- vars
+         +- foo.groovy          # for global 'foo' variable/function 
+         +- foo.txt             # help for 'foo' variable/function
 
 The `src` directory should look like standard Java source directory structure.
-This directory is added to the classpath when executing workflows. The groovy
-source files in this directory get the same sandbox / CPS transformation
-just like your workflow scripts.
+This directory is added to the classpath when executing workflows.
+
+The `vars` directory hosts scripts that define global variables accessible from
+workflow scripts.
+The basename of each `*.groovy` file should be a Groovy (~ Java) identifier, conventionally `camelCased`.
+The matching `*.txt`, if present, can contain documentation, processed through the system’s configured markup formatter
```

Comment:
This seems bad - as this can be changed independently and then this would affect the help rendering here.
I guess no different to all the job descriptions though.